Exporter/src/main/java/uk/ac/ebi/biostd/exporter/commons/FileUtils.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.ebi.biostd.exporter.commons;
+
+import java.nio.file.CopyOption;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Component;
+
+/**
+ * Wrappers class that encapsulate file operation to facilitate unit testing.
```

Comment:
Wrapper class that encapsulates file operations in order to facilitate unit testing